CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# dbt_hubspot v0.20.0
+[PR #150](https://github.com/fivetran/dbt_hubspot/pull/150) includes the following updates:
+
+## Breaking Change (`--full-refresh` required after upgrading)
```

Comment:
These breaking changes are occurring in `dbt_hubspot_source`. We should call that out here as Upstream Breaking Changes and link to the `dbt_hubspot_source` release notes or changelog. [You could use this PR you previously wrote for an upstream change](https://github.com/fivetran/dbt_greenhouse/pull/25/files) as an example for how to reword it.